### ISSUE TITLE
Specify filter range for numeric column

### ISSUE THREAD

**Issue by OhioGuy67** [andyperlitch/ngx-super-table#26]
Is there a way to easily specify a range for a numeric column (i.e., filter all values greater than 1st input AND all values less than 2nd input)?

**Comment by cmckni3** [comment]
Unforunately, this is not possible at the moment using the builtin [filters](https://github.com/andyperlitch/ngx-super-table/blob/master/src/lib/super-table-filters.ts#L81).

I like the idea and it is useful.

**Comment by jonyeezs** [comment]
Looking at the feature we could add a `NUMBE_RANGE` filter or other types.

Looks like this is extensible. @andyperlitch and @cmckni3  is this assumption right?

**Comment by cmckni3** [comment]
@jonyeezs, yes to have this built in would require changes. I believe a custom one can be specified in applications.
